.github/workflows/android-build.yml

```diff
@@ -68,6 +68,7 @@ jobs:
           ORT_NIGHTLY_VERSION=$(curl -s "${{ env.ORT_NIGHTLY_REST_API }}" | jq -r '.value[0].versions[0].normalizedVersion')
           echo "$ORT_NIGHTLY_VERSION"
           echo "ORT_NIGHTLY_VERSION=$ORT_NIGHTLY_VERSION" >> $GITHUB_ENV
+          $ONNXRUNTIME_VERSION = ${{ env.ORT_NIGHTLY_VERSION }}
```

Comment:
You can avoid making these changes. The version is >=1.19.0. And on CIs we insall 1.20.0.dev versions before installing the ort-genai python wheel.
So, it should be ok to not need these changes.

Reply:
Removed them